### PR TITLE
Sync translations with Transifex

### PR DESCRIPTION
**Pull request by patrys**
New languages:
* Arabic
* Armenian
* Bengali
* Danish
* Mongolian
* Swedish

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -5,14 +5,23 @@ host = https://www.transifex.com
 file_filter = saleor/dashboard/locale/<lang>/LC_MESSAGES/django.po
 source_file = saleor/dashboard/locale/en/LC_MESSAGES/django.po
 source_lang = en
+minimum_perc = 1
+trans.zh-Hans = saleor/dashboard/locale/zh_Hans/LC_MESSAGES/django.po
+trans.zh-Hant = saleor/dashboard/locale/zh_Hant/LC_MESSAGES/django.po
```
Comment (jxltom) [inline .tx/config]: I don't understand why only zh-hans/hanz are added here as a special case?

File: saleor/settings.py [hunk 2]
```diff
@@ -58,24 +58,30 @@ def get_bool_from_env(name, default_value):
...
+    ('bn', _('Bengali')),
     ('cs', _('Czech')),
+    ('da', _('Danish')),
     ('de', _('German')),
     ('en', _('English')),
     ('es', _('Spanish')),
     ('fa', _('Persian')),
     ('fr', _('French')),
     ('hu', _('Hungarian')),
+    ('hy', _('Armenian')),
```
Comment (jxltom) [inline saleor/settings.py]: It looks like Armenian is not valid language code in django

File: .tx/config (hunk 1 above)
Comment (patrys) [reply .tx/config]: They have a dash in their name that is translated to an underscore on our side.

File: saleor/settings.py (hunk 2 above)
Comment (patrys) [reply saleor/settings.py]: Good catch!

**Pull request merged**